pkgs/applications/misc/ape/apeclex.nix

```diff
@@ -0,0 +1,8 @@
+{ stdenv, attemptoClex, callPackage }:
+
+callPackage ./. {
+  pname = "ape-clex";
+  lexicon = "${attemptoClex}/clex_lexicon.pl";
+  description = "Parser for Attempto Controlled English (ACE) with a large lexicon (~100,000 entries)";
```

Comment:
why are these values not in default.nix?

Reply:
They are -- there are just two variations of the package, with the small lexicon and a larger one. Smaller lexicon is covered by LGPL, however, the larger one is GPL.